lib/patch-series.ts

```diff
@@ -288,7 +285,13 @@ export class PatchSeries {
         let coverLetterBody = prBody.trim();
 
         // parse the footers of the pullRequestDescription
-        const match = prBody.match(/^([^]+)\n\n([^]+)$/);
+        let match = prBody.match(/^([^]+)\n\n([^]+)$/);
+
+        if (!match && !prBody.match(/\n\n/)) {
+            // handle PR descriptions that have no body, just footers
+            match = prBody.match(/^()([A-Za-z]+: [^]+)$/);
```

Comment:
I think we need to allow dashes in footer names, like `Based-On:`. I.e. 
```suggestion
            match = prBody.match(/^()([-A-Za-z]+: [^]+)$/);
```

Reply:
Updated - sorry about that.  Added a test for based-on and it was not detected.  Updated the code to detect it.

However, this means based-on may not be well tested in the rest of the code.  I plan to add tests for PatchSeries, which will also test ProjectOptions but may add a separate ProjectOptions test.

Reply:
Thank you!